packages/server/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "main": "index.js",
   "scripts": {
+    "build": "electron-rebuild -o better-sqlite3",
```

Comment:
How come you didn't update the build-prod script to call and remove the need to update the build-binary script? Is there a reason to do it as a post-install when building the production binary?

Reply:
It seems like the rule might be: anything we do to the node_modules dependencies themselves needs to be done [here](https://github.com/cypress-io/cypress/blob/develop/scripts/binary/build.ts#L124). This includes patching packages and rebuilding native bindings.

Reply:
👍🏻 Makes sense. I assume you've found that via trial and error 😅